### DISCUSSION TITLE
/migrate-qcow2-images-from-kvm-to-vmware/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#12] [Announcements]
Migrate qcow2 images from KVM to VMWare
        
[https://blog.ktz.me/migrate-qcow2-images-from-kvm-to-vmware/](https://blog.ktz.me/migrate-qcow2-images-from-kvm-to-vmware/)

**Comment by JStMartin** [comment]
Thank you sir! I just needed part of what you did to run Amazon Linux on Apple silicon in VMWare.
